src/reducers/pause.js

```diff
@@ -169,10 +169,17 @@ function getSelectedFrame(state: OuterState) {
   return frames && frames.find(frame => frame.id == selectedFrameId);
 }
 
+// NOTE: currently only used for chrome
+function getScopes(state: OuterState) {
```

Comment:
I think this should be `getChromeScopes` so it is more obvious that is is only a single target method.